### DISCUSSION TITLE
完成支付宝支付，未升级成专业版

### DISCUSSION THREAD

**Discussion by bubulei** [getlantern/lantern-forum-cn#905] [General]
账号是：769132445@qq.com
![bbd55fe644ed86d2af9f2c2e357f4db](https://github.com/user-attachments/assets/d9ad2ff4-a201-423c-b962-0d0006452144)

**Comment by bubulei** [comment]
@Mike777Khoo 请帮忙处理下

**Comment by MaxMKMax** [comment]
@bubulei 您好. 我再帮您追问看看. 谢谢您的耐心等待.

**Reply by dreadeath1** [reply]
你好，我也遇到同样的问题，完成支付后未能升级成专业版，没有延长时长，请帮我核实处理一下

**Comment by bubulei** [comment]
有结果了吗

**Comment by bubulei** [comment]
@Mike777Khoo 请问什么时候能转成专业版啊

**Reply by YongboShen** [reply]
同问，看来是个系统问题。。。

**Comment by MaxMKMax** [comment]
@bubulei 您好. 问题已经解决了. 技术人员帮您升级成功了. 谢谢您的耐心等待.

**Reply by bubulei** [reply]
@Mike777Khoo 您好，可是我还是免费版 我需要重新下载吗

**Reply by bubulei** [reply]
还是没有升级成专业版

**Reply by Music802** [reply]
@Mike777Khoo    mrtos.jiang 也用同样问题，续费好多天了，但没有升级账号

**Comment by bubulei** [comment]
@Mike777Khoo 还是没有升级哎

**Reply by MaxMKMax** [reply]
@bubulei 我再帮您确认一下.

**Reply by greySkiesMarsh** [reply]
@bubulei Hello. I'm showing in the system that your account has been upgraded. Please follow the instructions in our FAQ page if you're having trouble authorizing your devices.

**Reply by bubulei** [reply]
好的 谢谢

**Comment by bubulei** [comment]
专业版网速很慢...其他网友是不是也是这样
